### PR TITLE
[th2-121] Verifies a message by the configured dictionary during enco…

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**
…ding

### CODE REVIEW

File: codec-fix/README.md [hunk 1]
```diff
@@ -49,5 +49,9 @@ E.g. original value: `100`, value after the removal of trailing zeroes: `1E+2`
 
 ## Release notes
 
++ 3.2.1
+  + Puts the BeginString(8) field during encode. The value to set is determined from the dictionary namespace.
+  + Verifies a message by the configured dictionary during encoding.
```
Comment (OptimumOpium) [inline codec-fix/README.md:54]: Shouldn't we add this information to the top-level README file?

Comment (Nikita-Smirnov-Exactpro) [reply codec-fix/README.md:54]: No, because we implemented it only for FIXCodec

**Review by OptimumOpium — APPROVED** [review]

File: codec-fix/README.md [hunk 2]
```diff
@@ -49,5 +49,9 @@ E.g. original value: `100`, value after the removal of trailing zeroes: `1E+2`
 
 ## Release notes
 
++ 3.2.1
+  + Puts the BeginString(8) field during encode. The value to set is determined from the dictionary namespace.
```
Comment (jean-michel-perli-sonsin) [inline codec-fix/README.md:53]: **Places** the BeginString(8) field during encode. The value to set **this field** is determined from the dictionary namespace.

**Review by jean-michel-perli-sonsin — COMMENTED** [review]
Hello, I just add a suggestion for one of the lines of Release notes. Please see if you agree with it and change it accordingly. The rest of the text is correct and well written.

**Pull request merged**